Header header
geometry_msgs/PoseStamped posePalm
leap_controller_capstone/FingerPoseStamped[] poseFingers
